lists-ui/src/views/Home/index.tsx

```diff
@@ -586,9 +586,19 @@ function Home() {
                   onChange={(value) => setPage(value - 1)}
                   total={totalPages || 9}
                   radius='md'
+                  siblings={2}
                   getControlProps={(control) => ({
                     'aria-label': `${control} page`,
                   })}
+                  getItemProps={(page) => {
```

Comment:
This was simplest way to acheive. Alternative was to implement a whole custom component with 5 x the amount of code. This seemed a reasonable short-cut as we only want to discourage deep pagination, not 100% block it.